unity/EVPreprocessing/Assets/Scripts/models_collection/ModelLayer.cs

```diff
@@ -3,11 +3,21 @@
 /* Information about layer.
  * It should be present in asset bundle for each GameObject representing a layer.
  */
+
+public enum DataType
+{
+    Mesh,
+    Volumetric
+}
+

```

Comment:
Same as above, we should be careful with the names and merge it with simulation flag in the future.